templates/terraform/examples/base_configs/example_backing_file.tf.erb

```diff
@@ -1,4 +1,7 @@
 <% autogen_exception -%>
+# This file has some scaffolding to make sure that names are unique when you
```

Comment:
It also has the default region/zone though. I think a more general name would be better, or separate that block out somewhere else.